scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.26.0] - 2018-11-08
+### Added
+- Support for job launch rate limits, from @scrosby
+- Pool info to usage command for CLI
+- Quota to cs usage CLI command.
+- Global launch rate limit
+### Changed
+- Data Locality service changed default port from 5000 to 35847
```

Comment:
Wasn't this only a local-dev facing change?

Reply:
Yes.